source/includes/_data.md

```diff
@@ -202,21 +203,25 @@ Optional.  Similar to customerFacingOrderNumber, but not customer facing.  A goo
 }
 ```
 
+### categoryName
+
+Optional.  Not visible except on the category sales report.  Overrides names that can be inferred from itemId.
+
 ### itemId
 
-Optional.  Identifier for an Instore menu item.  Enables more detailed reporting.
+Optional.  Identifier for an Instore menu item.  Enables more detailed reporting.  Will fill in categoryName when it is not supplied.
```

Comment:
Do you mean to say "Will fill in **itemName** when it is not supplied"?

Reply:
itemName is required, and this does fill in categoryName (the item has a single category so it's possible).  I see why it's confusing though...